pkg/virt-launcher/virtwrap/network/dhcpv6/dhcpv6.go

```diff
@@ -94,43 +94,49 @@ func (h *DHCPv6Handler) createConnection() (*FilteredConn, error) {
 func (h *DHCPv6Handler) ServeDHCPv6(conn net.PacketConn, peer net.Addr, m dhcpv6.DHCPv6) {
 	log.Log.V(4).Info("DHCPv6 serving a new request")
 
-	// TODO how can we make sure the request is from the vm? Is filtering requests arrived to the bridge interface enough?
+	// TODO if we extend the server to support bridge binding, we need to filter out non-vm requests
```

Comment:
Moved the comment to "virt-launcher: introduce dhcpv6" PR.
With masquerade binding the bridge shares the lan only with the vm. IIUC, there is no way the bridge will get a dhcp solicit from anywhere outside the vm.
However, with bridge binding, the pod interface is a slave of the bridge. Therefore, the bridge may get dhcp solicits not only from the vm.
dhcpv4 server filtered the requests according to the mac address. Only requests with the vnic mac address were handled. dhcpv6 cannot use the same logic since the mac address is not part of the dhcpv6 solicit request.

Reply:
> Moved the comment to "virt-launcher: introduce dhcpv6" PR. With masquerade binding the bridge shares the lan only with the vm. IIUC, there is no way the bridge will get a dhcp solicit from anywhere outside the vm. However, with bridge binding, the pod interface is a slave of the bridge. Therefore, the bridge may get dhcp solicits not only from the vm. dhcpv4 server filtered the requests according to the mac address. Only requests with the vnic mac address were handled. dhcpv6 cannot use the same logic since the mac address is not part of the dhcpv6 solicit request.
![image](https://user-images.githubusercontent.com/11764870/168980965-cf70e91b-5420-4d73-93d1-91d8004b6c27.png)
just in the picture，will the link-layer address be the filter criteria